.github/workflows/ruby-ci.yml

```diff
@@ -83,6 +83,141 @@ jobs:
           --fail
         if: failure()
 
+      - name: Login persistently
+        run: |
+          exe/gzr session login
+
+      - name: Create Connection
+        run: |
+          cat > Connection_faa_mysql.json <<HERE
```

Comment:
nice addition. pity you can't reuse your example script here, but I guess this is more granular for reporting the failure point

Reply:
Yeah, I could use the script but I wanted to break the steps out.